### ISSUE TITLE
The selected status of chart elements are not read by screen readers

### ISSUE THREAD

**Issue by spmealin** [julianna-langston/chartjs-plugin-a11y-legend#244]
When setting/clearing the selected status of chart elements in the legend area, the selected status of each element is not read by NVDA.

For example, if I have "bar1", "bar2", and "bar3", if I press space on "bar2", it should then prefix it with "not selected". I think you should be able to use ARIA to properly do this, but I have done absolutely no research.

**Comment by julianna-langston** [comment]
Fix released in v0.2.1. Please try it out and let me know if it works for you.
